src/redux/migrations.ts

```diff
@@ -1507,4 +1507,11 @@ export const migrations = {
       ),
     },
   }),
+  181: (state: any) => ({
+    ...state,
+    nfts: {
+      ...state.nfts,
+      nfts: [],
```

Comment:
we could update existing nfts in place, but we refetch all nfts on app start, so should be fine to just clean this